src/components/symptom-shark/presentational/LogEntry/LogEntry.tsx

```diff
@@ -42,18 +42,19 @@ export default function (props: SymptomSharkLogEntryProps) {
     let title: string = props.title ?? getDayOfWeek(props.date);
     let highlight = props.highlight ?? isToday(props.date);
     let subtitle = !props.title ? props.subtitle ?? getFullDateString(props.date) : undefined;
-    
+
     if (emptyLogEntry) {
         return <Card innerRef={props.innerRef} variant={highlight ? "highlight" : "subtle"} className="mdhui-ss-log-entry">
+            <ShinyOverlay />
```

Comment:
Do these little counter/severity widgets meet the accessibility guidelines? The size and background make me wonder.
 
<img width="118" height="67" alt="image" src="https://github.com/user-attachments/assets/eeaf7cc4-a5ee-49b6-a664-fc81e4e13f0d" />

Reply:
Yeah I mean there's a certain point at which accessibility / data density for usability clash, so likely not, but addressing this is not really in scope and would require actual design changes